demos/keyboard-a11y/js/five-star.js

```diff
@@ -84,25 +84,28 @@ var demo = demo || {};
             onCreate: [{
                 "this": "{that}.stars",
                 method: "mouseover",
-                args: {
-                    expander: {
-                        funcName: "demo.fiveStar.makeStarHandler",
-                        args: ["{that}", "{that}.hoverStars"]
+                args: [
```

Comment:
This change doesn't look right - you've changed the args list but left the method as "mouseover"

Reply:
You're right. Thanks for catching that. I've pushed up a fix.